LLDP_1.0.1-ts.241213.1800+schema/LLDP_1.0.1-ts.241213.1800+yaml.yaml

```diff
@@ -0,0 +1,623 @@
+    - tx-credit-max
+    - tx-fast-init
+    - notification-interval
+    - port-list
+  properties:
+    local-system-data:
+      type: object
+      required:
+        - chassis-id-subtype
+        - chassis-id
+        - system-name
+        - system-description
+        - system-capabilities-supported
+        - system-capabilities-enabled
+        - switching-protocol-on-off-is-avail
+        - supported-versions-of-lldp-list
+        - supported-kinds-of-lldp-agent-list
+      properties:
+        chassis-id-subtype:
+          type: string
+        chassis-id:
+          type: string
+        system-name:
+          type: string
+        system-description:
+          type: string
+        system-capabilities-supported:
+          type: object
+          required:
+            - other
+            - repeater
+            - bridge
+            - wlan-access-point
+            - router
+            - telephone
+            - docsis-cable-device
+            - station-only
+            - cvlan-component
+            - svlan-component
+            - two-port-mac-relay
+          properties:
+            other:
+              type: string
+            repeater:
+              type: string
+            bridge:
+              type: string
+            wlan-access-point:
+              type: string
+            router:
+              type: string
+            telephone:
+              type: string
+            docsis-cable-device:
+              type: string
+            station-only:
+              type: string
+            cvlan-component:
+              type: string
+            svlan-component:
+              type: string
+            two-port-mac-relay:
+              type: string
+        system-capabilities-enabled:
+          type: object
+          required:
+            - other
+            - repeater
+            - bridge
+            - wlan-access-point
+            - router
+            - telephone
+            - docsis-cable-device
+            - station-only
+            - cvlan-component
+            - svlan-component
+            - two-port-mac-relay
+          properties:
+            other:
+              type: string
+            repeater:
+              type: string
+            bridge:
+              type: string
+            wlan-access-point:
+              type: string
+            router:
+              type: string
+            telephone:
+              type: string
+            docsis-cable-device:
+              type: string
+            station-only:
+              type: string
+            cvlan-component:
+              type: string
+            svlan-component:
+              type: string
+            two-port-mac-relay:
+              type: string
+        supported-versions-of-lldp-list:
+          type: array
+          minItems: 0
+          items:
+            type: string
+        supported-kinds-of-lldp-agent-list:
+          type: array
+          minItems: 0
+          items:
+            type: string
```

Comment:
Attribute is of type MacAddress.
These means that the string needs to have a specific format.

Please double check all string attributes (that are no enumeration) for format definition and add pattern statements with proper regex definitions (please check existing model+yaml.yaml files for existing regex definitions and re-use these instead of inventing new ones).